packages/dx-react-scheduler/docs/guides/grouping.md

```diff
@@ -11,16 +11,13 @@ Appointments can be grouped by [resources](../reference/resources.md). Group nam
 
 ## Basic Usage
 
-To group by resources, [configure them](./resources.md) first.
-
-Grouping is implemented in the [IntegratedGrouping](../reference/integrated-grouping.md) plugin. Add this and the [GroupingState](../reference/grouping-state.md) plugin to enable this feature.
+To group by resources, [configure them](./resources.md) first. Then, add the [GroupingState](../reference/grouping-state.md) and [IntegratedGrouping](../reference/integrated-grouping.md) plugins.
 
 `GroupingState` has a `grouping` property that accepts an array of resources to group by. The resource order in this array is important. For example, if the Room resource is first and Attendees second, appointments are grouped in that order. If the `grouping` property is unspecified, appointments are grouped by the [main resource](../reference/resources.md/#properties).
 
-Group names (resource titles) are displayed in a grouping panel. To show it, add the [GroupingPanel](../reference/grouping-panel.md) plugin.
-
-A single appointment can appear in multiple groups if it is assigned to [multiple instances of a resource](./resources.md/#single-and-multiple-instance-resources). For example, an appointment assigned to three instances appears in three groups. The appointment's color is inherited from a particular resource instance. The `members` resource from the following demo illustrates this case:
+Group names (resource titles) are displayed in a group panel. To show it, add the [GroupingPanel](../reference/grouping-panel.md) plugin.
```

Comment:
In the second sentence, is "it" the group panel?
If so, please change it to:
"To show the panel..."